### DISCUSSION TITLE
Concern on the Question on 'Instant confirmation' of zkSync

### DISCUSSION THREAD

**Discussion by hgky95** [Cyfrin/foundry-full-course-cu#1903] [Q&A] [answered]
Hi everyone,

Regarding 'Lesson 18: Making Your First Transaction On zkSync,' I simulated the following scenario:

I bridged Sepolia ETH from the Sepolia Network to zkSync Sepolia on the testnet. After about one minute, I received the Sepolia ETH token on zkSync Sepolia, even though the transaction on Ethereum was still processing (in the sending status). I understand this is due to 'Instant confirmation,' which enables a seamless user experience—users can immediately utilize the transferred assets for further transactions.

Following this, I tried to bridge zkSync Sepolia ETH back to Sepolia ETH but encountered a warning message: 'You will receive funds only after a 24-hour withdrawal delay.' Meanwhile, I also tried sending zkSync Sepolia ETH from Address A to Address B within zkSync Sepolia, and this transaction occurred immediately.

My concern is that the seamless user experience provided by Instant Confirmation by zkSync actually applies only within their network. This ensures they can rollback all related transactions if the Layer 1 transaction on Ethereum fails. However, if the action involves going outside zkSync Sepolia, we always need to wait 24 hours (Withdrawal delay) to perform the next action.

Am I understand this situation correctly?

Thank you

**Comment by EngrPips** [comment, marked as answer]
Hello @ hgky95, I think you understood it correctly. I need to look into ZKsync more myself, too.

**Reply by jeka0203** [reply]
Like

**Reply by EngrPips** [reply]
I am not sure how you mean.
